api/v1beta1/team_types.go

```diff
@@ -270,6 +274,7 @@ const (
 	TeamUsedUpdated                       TeamConditionType = "TeamUsedUpdated"
 	TeamRequiredFieldsValidated           TeamConditionType = "TeamRequiredFieldsValidated"
 	TeamActiveEnvironmentDelete           TeamConditionType = "TeamActiveEnvironmentDelete"
```

Comment:
Can we rename this also `TeamActiveEnvironmentDeleted`?
Would it affect the current samsahai?

Reply:
should not affect ka